### PR TITLE
{lang}[GCCcore/10.3.0] CCL v1.12.1, SBCL v2.2.1

### PR DESCRIPTION
**Pull request by lexming**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by lexming** [comment]
@boegelbot: please test @ generoso

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node305.hydra.os - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/542d4cb0952225fde28ffdb520916b8d for a full test report.

File: easybuild/easyconfigs/c/CCL/CCL-1.12.1-GCCcore-10.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+toolchain = {'name': 'GCCcore', 'version': '10.3.0'}
+
+source_urls = ['https://github.com/Clozure/ccl/releases/download/v%(version)s']
+sources = ['%(namelower)s-%(version)s-linuxx86.tar.gz']
+checksums = ['923f2979144e96306cf5124800444e1c5cc0295fa9babb3ffc47a5631344b0aa']
+
+builddependencies = [
+    ('binutils', '2.36.1'),
+]
```
Comment (bedroge) [inline easybuild/easyconfigs/c/CCL/CCL-1.12.1-GCCcore-10.3.0.eb:19]: Could you also add M4 as build dependency (see https://github.com/easybuilders/easybuild-easyconfigs/pull/15075)?
```suggestion
    ('M4', '1.4.18'),
]
```

**Review by bedroge — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/c/CCL/CCL-1.12.1-GCCcore-10.3.0.eb (hunk 1 above)
Comment (lexming) [reply easybuild/easyconfigs/c/CCL/CCL-1.12.1-GCCcore-10.3.0.eb:20]: sure, thanks for the heads up on that issue

**Comment by lexming** [comment]
@boegelbot: please test @ generoso

**Review by bedroge — APPROVED** [review]
lgtm

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
pg-interactive.hpc.rug.nl - Linux UNKNOWN UNKNOWN, x86_64, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 3.6.8
See https://gist.github.com/012bb19633d5f56902a0d830f3c1c3fa for a full test report.

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
bob-Latitude-5300 - Linux Ubuntu 20.04, x86_64, Intel(R) Core(TM) i7-8665U CPU @ 1.90GHz, Python 3.8.10
See https://gist.github.com/4438fda06bb636c4acebb40c09c9e35f for a full test report.

**Comment by bedroge** [comment]
Going in, thanks @lexming!

**Pull request merged**